libs/barista-components/secondary-nav/src/section/secondary-nav-group.ts

```diff
@@ -35,7 +39,20 @@ import {
   changeDetection: ChangeDetectionStrategy.OnPush,
   encapsulation: ViewEncapsulation.Emulated,
 })
-export class DtSecondaryNavGroup {
-  /** The text value of the group header */
+export class DtSecondaryNavGroup implements AfterContentInit {
+  /** The text value of the group header. */
   @Input() label: string;
+
+  /** Whether or not the group is active. */
+  active = false;
+
+  /** @internal Active link directive within the group. */
+  @ContentChildren(DtSecondaryNavLinkActive)
+  private _links!: QueryList<DtSecondaryNavLinkActive>;
```

Comment:
I don't think we will need this.
```suggestion
  private _links: QueryList<DtSecondaryNavLinkActive>;
```